### ISSUE TITLE
Health sensor grenade in butt won't go off once in crit.

### ISSUE THREAD

**Issue by neetdev** [HippieStationArchive/HippieStation13-1#3115]
Made a grenade of 100u water 100u potassium:
-Put the health sensor+igniter in a grenade casing
-Made sure it was set to ON
-Proceeded to put it in my butt. 
-Got killed by traitor
-Grenade did not go off.

**Comment by Pyko1** [comment]
Are you sure that the grenade was secured and primed as well?

**Comment by neetdev** [comment]
Yes, after putting the two beakers I used the screwdriver to lock secure it and then clicked on it to see the menu with "Track health = OFF" (or something similar) and turned it to ON.

**Comment by JohnGinnane** [comment]
I wonder if your consumed the contents, as I know putting a beaker of anything (or food or pills) in your ass while mean you consume it as if you ate it, last I checked

**Comment by Scinexx** [comment]
I tried this a couple days ago and had the same results
